usersync/cookie_test.go

```diff
@@ -372,7 +395,7 @@ func TestTrimCookiesClosestExpirationDates(t *testing.T) {
 			actualKeys = append(actualKeys, key)
 		}
 
-		assert.ElementsMatch(t, testCases[i].expKeys, actualKeys, "[Test %d]", i+1)
+		assert.Len(t, actualKeys, testCases[i].expNumOfKeys, "[Test %d]", i+1)
```

Comment:
What if the lenght of `processedCookie.uids` gets directly asserted? We could reduce the code a bit further I think:
```
390       for i := range testCases {
391           processedCookie := writeThenRead(cookieToSend, testCases[i].maxCookieSize)
392  
393 -         actualKeys := make([]string, 0, 7)
394 -         for key := range processedCookie.uids {
395 -             actualKeys = append(actualKeys, key)
396 -         }
397 -  
398 -         assert.Len(t, actualKeys, testCases[i].expNumOfKeys, "[Test %d]", i+1)
    +         assert.Len(t, processedCookie.uids, testCases[i].expNumOfKeys, "[Test %d]", i+1)
399       }
400   }
usersync/cookie_test.go
```

Reply:
Have reverted this change based on Scott's comment above